mchf-eclipse/support/hex2dfu/hex2dfu.py

```diff
@@ -64,13 +64,13 @@ def save_dfu(ih):
 
     # Image element
     #
-    image_data = ih.tobinstr()
+    image_data = ih.tobinarray()
```

Comment:
Please add this, it does produce identical output on both python2 and python3:
```
    if len(image_data) != len(bytes(image_data)):
        #python 2 hack
        image_data = ih.tobinstr()

    data = struct.pack(
        "<II",
        ih.minaddr(),       # dwElementAddress
        len(image_data)     # dwElementSize
    ) + bytes(image_data)          # Data
```

Reply:
sure thing. einen Moment bitte  :)